content/en/docs/instrumentation/net/getting-started.md

```diff
@@ -21,8 +21,7 @@ Ensure that you have the following installed locally:
 
 The following example uses a basic
 [Minimal API with ASP.NET Core](https://learn.microsoft.com/aspnet/core/tutorials/min-web-api)
-application. If you are not using ASP.NET Core, that's OK — you can still use
-OpenTelemetry .NET Automatic Instrumentation.
+application.
```

Comment:
```suggestion
application. If you are not using a minimal API with ASP.NET Core, that's OK — you can use OpenTelemetry .NET with other frameworks as well. For a complete list of libraries for supported frameworks, see the [registry](https://deploy-preview-3364--opentelemetry.netlify.app/ecosystem/registry/?component=instrumentation&language=dotnet).
```
Similar to other language getting started guides, point out that other setups / frameworks should work as well and that they can take a look into the registry to find them